ivy_tests/test_ivy/test_frontends/test_numpy/test_ndarray/test_ndarray.py

```diff
@@ -364,3 +364,61 @@ def test_numpy_instance_argsort(
         x=x[0],
         axis=axis,
     )
+
+    
+@handle_cmd_line_args
+@given(
+    dtype_x_axis=helpers.dtype_values_axis(
+        available_dtypes=helpers.get_dtypes("numeric"),
+        min_axis=-1,
+        max_axis=0,
+        min_num_dims=1,
+        force_int_axis=True,
+    ),
+    num_positional_args=helpers.num_positional_args(
+        fn_name="ivy.functional.frontends.numpy.ndarray.prod"
+    ),
+)
+def test_numpy_ndarray_prod(
+    dtype_x_axis,
+    dtype,
+    out,
+    keepdims,
+    initial,
+    where,
+    as_variable,
+    num_positional_args,
+    native_array,
+):
+    input_dtype, x, axis = dtype_x_axis
+    where, as_variable, native_array = np_frontend_helpers.handle_where_and_array_bools(
+        dtype=dtype, out=out, keepdims=keepdims, initial=initial,
```

Comment:
please remove this line, the arguments below this line are the only needed ones

Reply:
@juliagsy done plz check this!!!